misc/language-tour-guide/src/components/CodeEditor.tsx

```diff
@@ -0,0 +1,40 @@
+import CodeMirror from "@uiw/react-codemirror";
+import { useState, useCallback } from "react";
+import { elixir } from "codemirror-lang-elixir";
+
+function CodeEditor() {
+  const [value, setValue] = useState(`
+defmodule EmployeeApp do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias EmployeeApp.Repo
+
+  schema "employees" do
+    field :name, :string
+    field :email, :string
+    field :department, :string
+    field :status, :string, default: "active"
+    timestamps()
+  end
+
+end
```

Comment:
I'd go with something simpler as a example, I think we never tested if/how ecto works with Popcorn.